@Types/ITechRecord.d.ts

```diff
@@ -42,11 +45,12 @@ export default interface ITechRecord {
   vehicleType: string;
   vehicleSize: string;
   vehicleConfiguration: string;
-  euroStandard?: string,
-  adrDetails?: AdrDetails,
+  euroStandard?: string;
+  adrDetails?: AdrDetails;
   brakes: {
     brakeCode: string,
     brakeCodeOriginal: string,
+    dtpNumber: string;
```

Comment:
This should have a coma

Reply:
If you take a look at typescript docs, all the attributes in the interfaces are delimited by semicolon (;) even though comma separation is not wrong. When I added new interfaces and attributes to existing interfaces I used the semicolon approach. I can change it to be consistent, but I'd rather do it in the Trailer tickets as they are still in techQA, this is in ready for amazing.

Reply:
Yep, that's fine for me!